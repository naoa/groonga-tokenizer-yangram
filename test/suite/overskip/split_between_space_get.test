register tokenizers/yangram

yangram_register 2 --split_symbol 1 --split_alpha 1 --overlap_skip 1
tokenize TokenYaBigramSplitSymbolAlphaSkipOverlap "This is" NormalizerAuto --mode GET
