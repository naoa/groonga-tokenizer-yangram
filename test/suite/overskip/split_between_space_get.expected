register tokenizers/yangram
[[0,0.0,0.0],true]
tokenize TokenYaBigramSplitSymbolAlphaOverskip "This is" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "th",
      "position": 0
    },
    {
      "value": "is",
      "position": 2
    },
    {
      "value": "is",
      "position": 4
    }
  ]
]
